<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>393</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PID控制</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>877</width>
     <height>336</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>877</width>
     <height>336</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>877</width>
     <height>336</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <widget class="QSlider" name="data_1_value_Qslider">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>20000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>串口号:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>12</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>标识符:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="data_1_Type_Flag_TextEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="data_1_enable_CheckBox">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>80</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>使能</string>
    </property>
   </widget>
   <widget class="QLabel" name="data_1_value_QLabel">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Min:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="data_1_Min_TextEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="data_1_Max_TextEdit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Max:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Max:</string>
    </property>
   </widget>
   <widget class="QSlider" name="data_2_value_Qslider">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>20000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>标识符:</string>
    </property>
   </widget>
   <widget class="QLabel" name="data_2_value_QLabel">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>120</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Min:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="data_2_Min_TextEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="data_2_Type_Flag_TextEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="data_2_enable_CheckBox">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>120</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>使能</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="data_2_Max_TextEdit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>120</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Max:</string>
    </property>
   </widget>
   <widget class="QSlider" name="data_3_value_Qslider">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>20000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>标识符:</string>
    </property>
   </widget>
   <widget class="QLabel" name="data_3_value_QLabel">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>160</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Min:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="data_3_Min_TextEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="data_3_Type_Flag_TextEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="data_3_enable_CheckBox">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>160</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>使能</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="data_3_Max_TextEdit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>160</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>200</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Max:</string>
    </property>
   </widget>
   <widget class="QSlider" name="data_4_value_Qslider">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>20000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>标识符:</string>
    </property>
   </widget>
   <widget class="QLabel" name="data_4_value_QLabel">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>200</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Min:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="data_4_Min_TextEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="data_4_Type_Flag_TextEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="data_4_enable_CheckBox">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>200</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>使能</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="data_4_Max_TextEdit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>200</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>240</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Max:</string>
    </property>
   </widget>
   <widget class="QSlider" name="data_5_value_Qslider">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>20000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>标识符:</string>
    </property>
   </widget>
   <widget class="QLabel" name="data_5_value_QLabel">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>240</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Min:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="data_5_Min_TextEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="data_5_Type_Flag_TextEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="data_5_enable_CheckBox">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>240</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>使能</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="data_5_Max_TextEdit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>240</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>280</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Max:</string>
    </property>
   </widget>
   <widget class="QSlider" name="data_6_value_Qslider">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>20000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>标识符:</string>
    </property>
   </widget>
   <widget class="QLabel" name="data_6_value_QLabel">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>280</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Min:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="data_6_Min_TextEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="data_6_Type_Flag_TextEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="data_6_enable_CheckBox">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>280</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>使能</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="data_6_Max_TextEdit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>280</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="Port_Number_comboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
